libp2p/multistream.nim

```diff
@@ -113,7 +113,7 @@ proc list*(m: MultistreamSelect,
 
 proc handle*(m: MultistreamSelect, conn: Connection) {.async, gcsafe.} =
   trace "handle: starting multistream handling"
-  tryAndWarn "multistream handle":
+  try:
```

Comment:
`tryAndWarm` did this nicely and in a standardized way support cancellation.
Improving `tryAndWarm` if something was missing was also possible.
This is going backward in time...